### DISCUSSION TITLE
Some problems when I installation of GiD plus

### DISCUSSION THREAD

**Discussion by dolamei** [Anura3D/Anura3D_OpenSource#243] [Q&A]
Dear developers,
I follow the manual which version is **Anura3D_TutorialManual_2024**, 
and when I in the step **2.4.2 The Anura3D 2023 problemtype**, 
I already pastsd the **Anura3D_2024.exe**  in the **Anura3D_2024.gid** file.

And then I also done the step **2.5 Installation of ParaView**,
but when I open my GiD plus (version 17),
it didn't show the same interface as in the manual is not displayed.

Mine↓
<img width="477" alt="螢幕擷取畫面 2024-07-27 000355" src="https://github.com/user-attachments/assets/71256f02-247e-4d92-802f-3e5823920fc3">

manual↓
<img width="304" alt="螢幕擷取畫面 2024-07-27 000450" src="https://github.com/user-attachments/assets/469494a8-d24c-421b-9828-1f85df6e1802">

### How can I try to solve this problem?
Thank you.

**Comment by FelipeMedan** [comment]
Hi dolamei, GiD always starts up as a general project, and you have to manually turn it into an Anura 3D type. If you go to Data>Problem type>Anura3D_2024, you should be able to see your interface as in the manual.

**Reply by dolamei** [reply]
I see, and I successfully solved this problem. Thank you.😁
